packages/zwave-js/src/lib/commandclass/EntryControlCC.ts

```diff
@@ -0,0 +1,683 @@
+			cc,
+			this.commandOptions,
+		);
+		if (response) {
+			return pick(response, ["keySupported"]);
+		}
+	}
+
+	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+	public async getEventSupported() {
+		this.assertSupportsCommand(
+			EntryControlCommand,
+			EntryControlCommand.EventSupportedGet,
+		);
+
+		const cc = new EntryControlCCEventSupportedGet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+		});
+		const response = await this.driver.sendCommand<EntryControlCCEventSupportedReport>(
+			cc,
+			this.commandOptions,
+		);
+		if (response) {
+			return pick(response, [
+				"dataTypeSupported",
+				"eventTypeSupported",
+				"keyCachedSizeMin",
+				"keyCachedSizeMax",
+				"keyCachedTimeoutMin",
+				"keyCachedTimeoutMax",
+			]);
+		}
+	}
+
+	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+	public async getConfiguration() {
+		this.assertSupportsCommand(
+			EntryControlCommand,
+			EntryControlCommand.ConfigurationGet,
+		);
+
+		const cc = new EntryControlCCConfigurationGet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+		});
+		const response = await this.driver.sendCommand<EntryControlCCConfigurationReport>(
+			cc,
+			this.commandOptions,
+		);
+		if (response) {
+			return pick(response, ["keyCachedSize", "keyCachedTimeout"]);
+		}
+	}
+
+	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+	public async setConfiguration(
+		keyCacheSize: number,
+		keyCacheTimeout: number,
+	) {
+		this.assertSupportsCommand(
+			EntryControlCommand,
+			EntryControlCommand.ConfigurationGet,
+		);
+
+		const cc = new EntryControlCCConfigurationSet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+			keyCacheSize,
+			keyCacheTimeout,
+		});
+		const response = await this.driver.sendCommand<EntryControlCCConfigurationReport>(
+			cc,
+			this.commandOptions,
+		);
+		if (response) {
+			return pick(response, ["keyCachedSize", "keyCachedTimeout"]);
+		}
+	}
+
+	protected [SET_VALUE]: SetValueImplementation = async (
+		{ property },
+		value,
+	): Promise<void> => {
+		if (property !== "keyCachedSize" && property !== "keyCachedTimeout") {
+			throwUnsupportedProperty(this.ccId, property);
+		}
+		if (typeof value !== "number") {
+			throwWrongValueType(this.ccId, property, "number", typeof value);
+		}
+		const valueDB = this.endpoint.getNodeUnsafe()!.valueDB;
+
+		let keyCachedSize = value;
+		let keyCachedTimeout: number = value;
+		if (property === "keyCachedSize") {
+			const oldKeyCachedTimeout = valueDB.getValue<number>(
+				getKeyCachedTimeoutStateValueID(this.endpoint.index),
+			);
+			if (oldKeyCachedTimeout == undefined) {
+				throw new ZWaveError(
+					`The "keyCachedSize" property cannot be changed before the key cached timeout is known!`,
+					ZWaveErrorCodes.Argument_Invalid,
+				);
+			}
```

Comment:
Can we assume/use a default timeout here? Otherwise the API user gets stuck in a loop